uniffi_bindgen/Cargo.toml

```diff
@@ -19,7 +19,7 @@ anyhow = "1"
 askama = { version = "0.11", default-features = false, features = ["config"] }
 cargo_metadata = "0.14"
 camino = "1.0.8"
-clap = { version = "3.1", features = ["cargo", "std", "derive"] }
+clap = { version = "~3.1", features = ["cargo", "std", "derive"] }
```

Comment:
This can now be reverted as deprecations are now off by default, see clap-rs/clap#3830 for details.  When you do start working through the deprecations as part of upgrading to 4.0, we've also improved the warnings for derive users.  See clap-rs/clap#3832 for examples.